pnpm-lock.yaml

```diff
@@ -1292,6 +1292,8 @@ importers:
       '@lowdefy/build': 4.0.0-rc.0
       '@lowdefy/client': 4.0.0-rc.0
       '@lowdefy/connection-axios-http': 4.0.0-rc.0
+      '@lowdefy/connection-mongodb': 4.0.0-rc.0
```

Comment:
Please revert this

Reply:
All the changes to the file?

Reply:
Changes made in [6dbcdb5](https://github.com/lowdefy/lowdefy/pull/1469/commits/6dbcdb58656dc5e6d004c7c6f14db5b52da6ddaf)